pygbm/splitting.py

```diff
@@ -76,6 +78,7 @@ def __init__(self, n_features, binned_features, n_bins,
         self.l2_regularization = l2_regularization
         self.min_hessian_to_split = min_hessian_to_split
         self.min_samples_leaf = min_samples_leaf
+        self.min_gain_to_split = min_gain_to_split
```

Comment:
You should raise a `ValueError` if the user passes `min_gain_to_split < 0`.

Reply:
There's already a check in the grower. BTW `min_gain_to_split` is only a parameter of the grower, not of `GradientBoostingMachine`. We should move it there right?

Reply:
Right this is fine for now. No need to expose it to the public API at this time.